docs/examples/shading/plot_far_shading_with_pvgis_horizon_data.py

```diff
@@ -0,0 +1,96 @@
+"""
+Far-Shading with PVGIS Horizon Data
+=========================
```

Comment:
```suggestion
Far-Shading with PVGIS Horizon Data
===================================
```

Sphinx complains when underline length doesn't match the title text length